### PR TITLE
GH-844: giving card content update

### PR DESCRIPTION
**Pull request by ksuhartono97**
change text in bank transfer card to clearly indicate ripple out remarks

### CODE REVIEW

File: ui/src/components/ripple-out/RippleOutGivingCard.js [hunk 1]
```diff
@@ -0,0 +1,217 @@
...
+            </Text>
+            <Text fontWeight="bold">Bank Code: 009</Text>
+            <Text fontWeight="bold">Branch Code: 845</Text>
+            <Text fontWeight="bold">
+              Account Name: Harvest Mission Community Church (Hong Kong) Limited
+            </Text>
+            <Text fontWeight="bold"> Account Number: 013012090</Text>
+            <Text fontWeight="bold">
+              Transfer Remarks: Please indicate the giving is for the “Ripple
+              Out Campaign”
```
Comment (tamarayustian) [inline ui/src/components/ripple-out/RippleOutGivingCard.js:67]: can add example? isnt it supposed to be Ripple Out: [Your Name]? lol it kept changing

File: ui/src/components/ripple-out/RippleOutGivingCard.js [hunk 2]
```diff
@@ -0,0 +1,217 @@
...
+            </Text>
+          </Box>
+
+          <Text fontWeight="bold">
+            If our account name is too long and exceeds the number of permitted
+            characters, you may shorten it as “Harvest Mission Community Church
+            HK”.
+          </Text>
+          <Text fontWeight="bold">
+            Please email a copy of your transfer receipt along with your Full
```
Comment (tamarayustian) [inline ui/src/components/ripple-out/RippleOutGivingCard.js:77]: still need to do this even when it's already indicated in transfer remarks?

File: ui/src/components/ripple-out/RippleOutGivingCard.js [hunk 3]
```diff
@@ -0,0 +1,217 @@
...
+          </Text>
+          <Text fontWeight="bold">P.O. Box 84282 HUNGHOM BAY POST OFFICE</Text>
+        </Box>
+      );
+    default:
+      return 'not detected';
+  }
+};
+
+export const RippleOutGivingCard = (cardinfo) => {
```
Comment (tamarayustian) [inline ui/src/components/ripple-out/RippleOutGivingCard.js:135]: camelCase for `cardInfo` here please to keep it consistent

File: ui/src/components/ripple-out/RippleOutGivingCard.js [hunk 4]
```diff
@@ -0,0 +1,217 @@
...
+  Modal,
+  ModalOverlay,
+  ModalContent,
+  ModalHeader,
+  ModalBody,
+  ModalCloseButton,
+  Center,
+  Image,
+  ModalFooter,
+} from '@chakra-ui/react';
```
Comment (tamarayustian) [inline ui/src/components/ripple-out/RippleOutGivingCard.js:22]: minor but can you add a new line here to make it neater haha

**Review by tamarayustian — CHANGES_REQUESTED** [review]

File: ui/src/components/ripple-out/RippleOutGivingCard.js (hunk 2 above)
Comment (ksuhartono97) [reply ui/src/components/ripple-out/RippleOutGivingCard.js:77]: yes bank transfer should just do this

**Review by tamarayustian — APPROVED** [review]

**Pull request merged**